classes/data-stores/ActionScheduler_wpPostStore.php

```diff
@@ -470,7 +470,10 @@ public function delete_action( $action_id ) {
 			throw new InvalidArgumentException(sprintf(__('Unidentified action %s', 'action-scheduler'), $action_id));
 		}
 		do_action( 'action_scheduler_deleted_action', $action_id );
-		wp_delete_post($action_id, TRUE);
+
+		if ( ! wp_delete_post( $action_id, TRUE ) ) {
+			$this->mark_migrated( $action_id );
```

Comment:
@rrennick do we want to mark this migrated here? It could mean the status is changed to migrated when non-migration related activities occur. Especially now that we may have `wpPostStore` running for a while after updating to 3.0 if an action with args > 191 chars is found, we can't assume migration will happen soon after updating anymore.

Reply:
@thenbrent Good catch. I updated the PR to add a check in the action migrator.

Reply:
Checking for a null action is a good approach. 👍